scss/6-components/_targeted-content.scss

```diff
@@ -79,18 +79,3 @@
     }
   }
 }
-
-.cads-targeted-content__expand-all {
```

Comment:
This was included here but the actual code for this was in public website. Still not sure exactly which side this should live on, but we need to at least pick _a_ side!